packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -39,6 +40,7 @@ export const MenuList = compose<MenuListType>({
   },
   useRender: (userProps: MenuListProps, useSlots: UseSlots<MenuListType>) => {
     const menuList = useMenuList(userProps);
+    const context = useMenuContext();
     const contextValue = useMenuListContextValue(menuList);
```

Comment:
nit, rename these maybe so they are more clear?
```suggestion
    const menuContext = useMenuContext();
    const menuListContextValue = useMenuListContextValue(menuList);
```